src/components/_inputs/InputEnum.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <a-form-item :label="schema.title">
+    <a-select :defaultValue="schema.default">
+      <a-select-option v-for="item in schema.enum" :value="item">
+        {{item}}
```

Comment:
Mostrar o `item` aqui entregaria para o usuário um `apply_at` por exemplo, não é boa ideia não, é necessário usar um objeto do i18n (tipo [`i19DiscountType`](https://github.com/ecomclub/i18n/blob/master/src/en_us/index.js#L591)).

Como este componente é genérico você pode receber este objeto via prop (`i19Enum` seria um prop), faz sentido?

```suggestion
        {{ i19Enum ? i19Enum(item) : item }}
```